_includes/delete-token.md

```diff
@@ -1,45 +0,0 @@
-{% assign documentation_section = include.documentation_section %}
```

Comment:
Should `delete-token.md` be deleted?

Reply:
No! I deleted it by mistake. I commited dx-1171 earlier today where I changed something in it again, so I hope I can fix it in a merge conflict?

Reply:
You can just check it out from `develop` again by doing the following on the command line:

```bash
git checkout develop -- _includes/delete-token.md
```

That will bring the file back, so you can commit it again. The ideal would be to change the commit where you removed it so it isn't removed, but I think we need to do that on a call.